packages/remark-math/package.json

```diff
@@ -32,9 +35,11 @@
   ],
   "scripts": {
     "prepare": "npm run build",
-    "build": "babel -d dist index.js inline.js block.js"
+    "build": "babel -d dist index.js inline.js block.js",
+    "build-tii": "babel -d build index.js inline.js block.js"
```

Comment:
This builds to the `build` directory instead of the `dist` directory, as that is where the artifactory docker image was expecting the code to be.